src/checkqueue.h

```diff
@@ -33,34 +30,29 @@ template <typename T>
 class CCheckQueue
 {
 private:
-    //! Mutex to protect the inner state
-    boost::mutex mutex;
-
-    //! Worker threads block on this when out of work
-    boost::condition_variable condWorker;
-
-    //! Master thread blocks on this when out of work
```

Comment:
629dae24827121412fb24d31ac19a918ade1db3f: why remove the documentation?

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/18710#issuecomment-619008172).